lib/matplotlib/quiver.py

```diff
@@ -520,11 +532,9 @@ def get_datalim(self, transData):
     @allow_rasterization
     def draw(self, renderer):
         self._init()
-        if (self._new_UV or self.angles == 'xy'
-                or self.scale_units in ['x', 'y', 'xy']):
-            verts = self._make_verts(self.U, self.V)
-            self.set_verts(verts, closed=False)
-            self._new_UV = False
+        verts = self._make_verts(self.U, self.V)
```

Comment:
Am I reading this right that this is just removing a path to short-circuit re-generating the verts? I think we have to drop this `if` statement because even in the case where you are specifying angles, the user could have changed the axis inversion sense the last draw so you _always_ have to regenerate the verts.